masterlist.yaml

```diff
@@ -25181,3 +25198,16 @@ plugins:
       - <<: *useOnlyOneX
         condition: 'many("(CFR-CloaksForRequiem|Requiem - Cloaks( of Skyrim| of Skyrim Hard - Times| of Skyrim Patch| WICCloaks CCFR USKP)?)\.esp")'
         subs: [ 'Requiem patch for Cloaks of Skyrim' ]
+  - name: 'Requiem - Immersive Jewelry( BTC| ISC)?\.esp'
```

Comment:
This regex would be slightly neater with the space outside the parentheses, but it's a minor point.